SEED-platform/seed
# Update inventory selection to include current inventory type only

#### Any background context you want to provide?
When a selection is made on the inventory list, an indicator appears in the top right corner of the table (`(25 selected...`). This selection indicator was double counting properties and tax lots regardless of the current inventory type.

#### What's this PR do?
Filters the selection count to include the current inventory type. This feature has been added to select all and partial select.

#### How should this be manually tested?
- View a cycle that has nested taxlots and select all. The total should match the current page inventory count. 
- Select a few properties and taxlots, the total should reflect only the relevant inventory type

#### What are the relevant tickets?
#3817

#### Screenshots (if appropriate)
NEW
![Screenshot 2023-02-14 at 1 48 42 PM](https://user-images.githubusercontent.com/58446472/218875790-6e1ba1db-e756-4f79-b432-8ef39149d6de.png)
![Screenshot 2023-02-14 at 1 49 01 PM](https://user-images.githubusercontent.com/58446472/218875786-9425f8fb-4b55-40e0-935a-6a12f7887e25.png)

OLD
![Screenshot 2023-02-14 at 1 49 33 PM](https://user-images.githubusercontent.com/58446472/218875784-1c32f961-303c-4270-b51d-15f1b5fefe07.png)
![Screenshot 2023-02-14 at 3 06 18 PM](https://user-images.githubusercontent.com/58446472/218875782-997a3469-3cd5-45dd-b22f-6795481da803.png)
